### ISSUE TITLE
fail build version.h not found when build more than 18threads (18 ok, 19 fails)

### ISSUE THREAD

**Issue by sl1pkn07** [flightlessmango/MangoHud#862]
i have 48threads machine

if try to build the project with all threads active fail build

~~~
In file included from ../src/amdgpu.cpp:7:
../src/overlay.h:10:10: fatal error: version.h: No such file or directory
   10 | #include "version.h"
      |          ^~~~~~~~~~~
~~~

if set jobs (-j) lower than 19, builds ok

i think the command for generate the `version.h` file should be created in configure time instead of build time

greetings

**Comment by flightlessmango** [comment]
> i think the command for generate the version.h file should be created in configure time instead of build time

But that's already the case?

**Comment by sl1pkn07** [comment]
no. "meson configure foo" as-sis not generated the file, but yes when do "meson compile foo"

**Comment by flightlessmango** [comment]
Do we have control over that? Seems to me like Meson chooses what is in setup and what is in compile

**Comment by evelikov** [comment]
Reportedly my PR wasn't enough to fix, so @sl1pkn07 feel free to reopen.

I'm happy to take another look, but will need the complete log (`meson setup` and `meson compile` aka `ninja`), alongside the contents of `build.ninja`. Thanks o/
